gitlab/pkg/apis/sources/v1alpha1/gitlabsource_types.go

```diff
@@ -171,6 +182,26 @@ func (s *GitLabSourceStatus) MarkSecret() {
 	gitLabSourceCondSet.Manage(s).MarkTrue(GitLabSourceConditionSecretProvided)
 }
 
+// MarkWebhook sets the condition that the source has set its webhook configured.
```

Comment:
I was confused when I first read that sentence. How about just
> MarkWebhook sets the WebhookConfigured condition to True.

since you already gave a good explanation of what the condition means above?